### PR TITLE
fixed -- data : KaKashi1210.json

### PR DESCRIPTION
**Pull request by KaKashi1210**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4709"></a>

### CODE REVIEW

**Review by Saadat-B — CHANGES_REQUESTED** [review]
Hey Aditya, please rename the `.json` file name and add your GitHub username exactly as it appears after `data/`. 
Please replace `AdityaDafe` with `KaKashi1210`
Please let us know if you need any help with it 🙂

File: data/AdityaDafe.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "name": "Aditya Dafe",
+  "type": "personal",
+  "bio": "**CS UnderGrad** && **Open soure Enthusiast** ",
+  "tags": [ "Open Source", "Tech"],
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AdityaDafe",
+      "icon": "FaTwitter"
+    }
```
Comment (priyankarpal) [inline data/AdityaDafe.json:11]: Add a comma 
``` suggestion
    },
```

File: data/AdityaDafe.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    {
+      "name":"Connect with me on LinkedIn",
+      "url":"https://www.linkedin.com/in/aditya-dafe-601a2a243/",
+      "icon":"FaLinkedIn"
```
Comment (priyankarpal) [inline data/AdityaDafe.json:15]: Change the icon to `SiLinkedin`
```suggestion
      "icon":"SiLinkedin"
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]
Rename your name from `data/AdityaDafe.json` to your GitHub username such as `data/KaKashi1210.json`.

**Comment by KaKashi1210** [comment]
ok guys thanks for the help imma fix it asap

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

**Review by KaKashi1210 — COMMENTED** [review]
Thanks for the help !!

File: data/KaKashi1210.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AdityaDafe",
+      "icon": "FaTwitter"
+    },
+    {
+      "name":"Connect with me on LinkedIn",
+      "url":"https://www.linkedin.com/in/aditya-dafe-601a2a243/",
+      "icon":"SiLinkedin"
+    }
+
```
Comment (krshkun) [inline data/KaKashi1210.json:17]: Looks great! Empty line will be fixed by github actions

**Review by krshkun — APPROVED** [review]

**Pull request merged**